### ISSUE TITLE
[bug] (Linux) CLI:  Not possible to make OpenVPN connection without defining a custom DNS

### ISSUE THREAD

**Issue by stenya** [ivpn/desktop-app#196]
# Bug report

## Describe your environment

* OS name and version: Linux
* IVPN app version: 3.9.8  

## Describe the problem
Not possible to make OpenVPN connection without defining a custom DNS
The issue is related to: Linux + CLI + OpenVPN

### Steps to reproduce:

1. ivpn dns -off
2. ivpn connect -p ovpn amsterdam

### Observed Results:

```
[OpenVPN] Connecting to: Amsterdam, NL (Netherlands) nl.gw.ivpn.net UDP:2049...
Connecting...
Connected but failed to initialize custom DNS!
Disconnecting...
Error: failed to initialize custom DNS!
```

### Expected Results:

```
[OpenVPN] Connecting to: Amsterdam, NL (Netherlands) nl.gw.ivpn.net UDP:2049...
Connecting...
VPN                    : CONNECTED
```

**Comment by stenya** [comment]
@gorkapernas  v3.9.9 (base package) is ready for testing.

**Comment by gorkapernas** [comment]
Verified fixed on version 3.9.9/3.9.10, tested on Ubuntu 20.0.4, Fedora 33 and Fedora 35, the issue no longer occurs, it is possible to establish connection in the CLI with OpenVPN AND without setting up a custom DNS.

**Comment by stenya** [comment]
v3.9.14
